Gemfile

```diff
@@ -2,6 +2,14 @@ source 'https://rubygems.org'
 
 gemspec
 
-group :test do
+group :development, :test do
   gem 'between_meals', :git => 'https://github.com/facebook/between-meals'
+  gem 'chef-dk', '~> 3.13.0'
```

Comment:
I wonder if we should split these up between test and development, but this is good enough for now.

Reply:
yeah, I thought about that but wasn't entirely sure what I'd isolate so I figured I'd start here. If you have an opinion I'm happy to do it.